go/chat/storage/version.go

```diff
@@ -18,6 +18,7 @@ type ServerVersions struct {
 }
 
 func NewServerVersions(g *globals.Context) *ServerVersions {
+	initLocksRepoOnce(g)
```

Comment:
Can you do this instead?

https://golang.org/doc/effective_go.html#init

Reply:
fixed